### DISCUSSION TITLE
Send message as HTML through applications API

### DISCUSSION THREAD

**Discussion by inakibx** [chatwoot/chatwoot#10312] [Chatwoot APIs] [answered]
Hello! I am trying to send a message through the application's API as if it was an email by calling the "Create New Message" endpoint ([see reference in API doc](https://www.chatwoot.com/developers/api/#tag/Messages/operation/create-a-new-message-in-a-conversation)). 

The message is sent, but is there a way to pass the content as HTML? Is there also a way to set a subject for the email sent?

**Comment by sojan-official** [comment, marked as answer]
We don't support html, you can use markdown. 

for email subject ref: https://discord.com/channels/647412545203994635/1296423979287969835/1296629554374250559

>  you can look into the networks tab of browser and replicate the same payload 
> 
> Example : 
> 
> message[content]: hello
> inbox_id: 25
> contact_id: 25
> source_id: test@test.com
> additional_attributes[mail_subject]: hello
> assignee_id: 1
> 
> 
> additional_attributes[mail_subject] -> take care of the email subject

**Comment by adriaandotcom** [comment]
For me that didn't work. You can use something like this. Use this payload:

https://github.com/chatwoot/chatwoot/blob/master/app/javascript/dashboard/components-next/message/fixtures/emailConversation.js

```js
  const url = new URL(
    `/api/v1/accounts/${CHATWOOT_ACCOUNT_ID}/conversations/${conversationId}/messages`,
    CHATWOOT_URL
  );

  const body = {
    message_type: 0,
    content_type: "incoming_email",
    content_attributes: {
      email: {
        from: [email],
        cc,
        subject: subject || "",
        text_content: {
          full: content,
        },
      },
      cc_email: cc,
    },
    content,
    sender: {
      email,
      
    },
  };

  const response = await fetch(url, {
    headers: applicationHeaders,
    method: "POST",
    body: JSON.stringify(body),
  });
```
